src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -434,4 +434,36 @@
     <value>{0} {1}
     The default value '{1}' is not a valid value for {0}.</value>
   </data>
+  <data name="AddRefPostActionFailed" xml:space="preserve">
+    <value>Failed to add reference {0} to project file {1}</value>
+  </data>
+  <data name="AddRefPostActionMultipleProjFiles" xml:space="preserve">
+    <value>Found multiple possible project files in the same directory. Could not determine which to add a reference to. Files:</value>
+  </data>
+  <data name="AddRefPostActionNoProjFileFound" xml:space="preserve">
+    <value>Unable to add reference {0}. Could not find an appropriate project file.</value>
```

Comment:
In AddRefPostActionFailed you use "Failed to add...", but "Unable to add..." here. We should be consistent. I think "Unable to add..." is better.

Also I think "Could not find an apporpriate project file" would be better as "Could not find project file".